db/changes.go

```diff
@@ -347,7 +347,7 @@ func (db *DatabaseCollectionWithUser) wasDocInChannelPriorToRevocation(ctx conte
 	// Iterate over the channel history information on the document and find any periods where the doc was in the
 	// channel and the channel was accessible by the user
 	for _, docHistoryEntry := range append(syncData.ChannelSet, syncData.ChannelSetHistory...) {
-		if docHistoryEntry.Name != chanName {
+		if docHistoryEntry.Name != chanName && "*" != chanName {
```

Comment:
I think this could be the `channels.UserStarChannel` const

Reply:
Addressed in [43121ce](https://github.com/couchbase/sync_gateway/pull/5961/commits/43121ce71726f4b6e6f8e21b059c40fbac8ebf1b)